WonderInventar.py

```diff
@@ -0,0 +1,40 @@
+from WonderTexte import *
+from WonderBlumen import *
+from WonderWoerter import *
+
+class WonderInventar(object):
+    def __init__(self, wortvergleicher):
+        self.inventar = []
+        self.wortvergleicher = wortvergleicher
+        self.text =""
+
+    def SteckeBlumeInsInventar(self,blume:WonderBlume):
+        self.inventar.append(blume)
+        return "Du hast die Blume nun in deiner Tasche. Versuche nun deine Tasche anzusehen!"
+ 
+    def zaehleRiesigeBlumen(self):
+        zaehler=0
+        for blume in self.inventar:
+            if blume.ermittleObRiesig():
+                zaehler = zaehler+1
+        return zaehler
```

Comment:
Die Lösung ist nicht falsch, ich wollte dir an der Stelle nur erzählen, dass python einen sehr eigenen Konstrukt für so was hat: List Comprehensions https://www.w3schools.com/python/python_lists_comprehension.asp
Ich weiß ehrlich gesagt nicht, ob sie effizienter sind intern und bei den kleinen Objekten, mit denen wir hier zu tun haben, ist es eh egal. Mit einer List Comprehension wird diese Methode folgendermaßen aussehen:
```suggestion
        riesigeBlumen = [blume for blume in self.inventar if blume.ermittleObRiesig()]
        return len(riesigeBlumen)
```